src/samplers/cpu/linux/usage/proc_stat.rs

```diff
@@ -118,18 +134,33 @@ impl ProcStat {
             let header = parts.first().unwrap();
 
             if *header == "cpu" {
-                for (field, counter) in self.counters_total.iter_mut().enumerate() {
+                let mut busy: u64 = 0;
+
+                for (field, counter) in self.total_counters.iter_mut().enumerate() {
                     if let Some(Ok(v)) = parts.get(field + 1).map(|v| v.parse::<u64>()) {
-                        counter.set(elapsed, v.wrapping_mul(self.nanos_per_tick))
+                        if field < 2 || field > 4 {
```

Comment:
I think we only want to skip `idle` and `io_wait` so should be `field < 3`? Also, should we drop a comment in as to what the magic numbers correspond to?

Reply:
Changed to use comparison to two consts to make this clearer.